utils/istio/istio.go

```diff
@@ -64,19 +64,42 @@ func GetVirtualServiceNamespaceName(vsv string) (string, string) {
 	return namespace, name
 }
 
+func MultipleVirtualServiceConfigured(rollout *v1alpha1.Rollout) bool {
+	if rollout.Spec.Strategy.Canary.TrafficRouting.Istio.VirtualServices != nil {
```

Comment:
how about changing to `return rollout.Spec.Strategy.Canary.TrafficRouting.Istio.VirtualServices != nil`

Reply:
Updated the PR